stdlib/3/dbm/__init__.pyi

```diff
@@ -1,11 +1,29 @@
-
-from typing import Union, MutableMapping
+import sys
+from types import TracebackType
+from typing import Iterator, Optional, Type, Union, MutableMapping
 
 _KeyType = Union[str, bytes]
 _ValueType = Union[str, bytes]
 
+if sys.version_info >= (3, 8):
+    from typing import Literal
+    _Flag = Literal['r', 'w', 'c', 'n']
+else:
+    _Flag = str
```

Comment:
Just replace this with:

```suggestion
from typing_extensions import Literal
_Flag = Literal['r', 'w', 'c', 'n']
```

This will hopefully also fix the pytype error. You could also inline `_Flag` if you are using this import.